README.md

```diff
@@ -137,3 +147,30 @@ https://docs.python.org/2/library/logging.html#logging-levels.
 
 `log` is a log object created using python's standard `logging` module. You can
 read more about it at https://docs.python.org/2/library/logging.html.
+
+### Metric-Grouping
+We can collect the Metrics based on time interval.
```

Comment:
@Rajendra-R Something like "When writing a log using self.log, you can specify type=metric. If this is done, a background thread will automatically group multiple such lines into one by averaging values (to prevent writing too many log lines)"